contrib/PKGBUILD

```diff
@@ -0,0 +1,35 @@
+# Maintainer: Bruno Pagani (a.k.a. ArchangeGabriel) <archange@archlinux.org>
+# Contributor: Mirco Tischler <mt-ml at gmx dot de>
+
+pkgname=fwupd
+pkgver=LATEST
+pkgrel=1
+pkgdesc='A simple daemon to allow session software to update firmware'
+arch=('i686' 'x86_64')
+url='https://github.com/hughsie/fwupd'
+license=('GPL2')
+depends=('appstream-glib' 'fwupdate' 'colord')
+#makedepends=('meson' 'valgrind' 'docbook2x' 'gobject-introspection' 'gtk-doc')
```

Comment:
Not really a debug line in fact, it’s for when we will be able to build with docbook2x again. But I’ve removed it from the file here, I’m keeping it on my side anyway.